### PR TITLE
Optimise-privado-api

### PR DESCRIPTION
**Pull request by aminlatifi**
Added privadoVerified field to the user entity

### CODE REVIEW

File: src/adapters/adaptersFactory.ts [hunk 1]
```diff
@@ -20,7 +20,6 @@ import { SuperFluidAdapterInterface } from './superFluid/superFluidAdapterInterf
 import { AbcLauncherAdapter } from './abcLauncher/abcLauncherAdapter';
 import { AbcLauncherMockAdapter } from './abcLauncher/abcLauncherMockAdapter';
 import { PrivadoAdapter } from './privado/privadoAdapter';
-import { IPrivadoAdapter } from './privado/privadoAdapterInterface';
```
Comment (ae2079) [inline src/adapters/adaptersFactory.ts:23]: It is OK to remove the Interface, but do we need to do that?

**Review by ae2079 — CHANGES_REQUESTED** [review]
Thanks @aminlatifi 
just need little changes

File: src/entities/user.ts [hunk 2]
```diff
@@ -207,6 +208,13 @@ export class User extends BaseEntity {
   @Column('integer', { array: true, default: [] })
   privadoVerifiedRequestIds: number[];
 
+  @Field(_type => Boolean, { nullable: true })
+  privadoVerified(): boolean {
```
[the remark this answers is not in the document]
Comment (aminlatifi) [reply src/entities/user.ts:212]: It's working already, 
I think that `get` is needed for internal use, the current format is fine with graphql

File: src/adapters/adaptersFactory.ts (hunk 1 above)
Comment (aminlatifi) [reply src/adapters/adaptersFactory.ts:23]: It's not used anymore. We don't use mock adapter.

File: src/entities/user.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (aminlatifi) [reply src/entities/user.ts:212]: As you see tests are passed.

**Review by ae2079 — APPROVED** [review]
LGTM
thanks

**Pull request merged**